sites/waterworld/server/templates/website-section/global-thought-leaders.marko

```diff
@@ -0,0 +1,129 @@
+import { getAsObject } from '@base-cms/object-path';
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/get-adunit';
+import { video, whitepaper, company } from '../../api/fragments/global-thought-leaders';
+
+$ const { site } = out.global;
+$ const page = {
+  title: 'Global Thought Leader Series',
+  description: 'Certain companies in the water and wastewater industry are veritable trailblazers, forging ahead and sharing their expertise with others in an effort to help them stay current with new trends and technologies and do their jobs more efficiently. Our new Global Thought Leader Series will feature a collection of white papers and related videos from these innovators, giving you free and easy access to a bank of useful knowledge.',
+};
+$ const imageOpts = {
+  fit: 'clip',
+  h: 100,
+  minH: 100,
+};
+$ const section = getAsObject(data, 'section');
+
+<theme-pennwell-website-section-layout section=section>
+  <@head>
+    <cms-ad-gam-head />
+  </@head>
+  <@above-container>
+    <endeavor-gam-ad-unit-define-display name="lb1" modifiers=["top-of-page"] />
+  </@above-container>
+
+  <div class="page-wrapper default-page">
+    <div class="row">
+      <div class="col">
+        <div class="page-wrapper__header">
+          <endeavor-published-content-breadcrumbs title=page.title />
+          <h1 class="page-wrapper__title">${page.title}</h1>
+          <if(page.description)>
+            <p class="page-wrapper__description">${page.description}</p>
+          </if>
+        </div>
+        <div class="page-wrapper__contents">
+          <hr>
+          <h5>Featured Videos</h5>
+          <cms-query-website-scheduled-content|{ nodes }|
+            section-id=section.id
+            limit=2
+            include-content-types=["Video"]
+            query-fragment=video
+          >
+            <div class="row">
+            <for|content| of=nodes>
+              <div class="col">
+                <endeavor-item>
+                  <@image
+                    image=content.company.primaryImage
+                    link-to=content.company
+                    modifiers=["center", "mh-100"]
```

Comment:
Again, use a modifier that describes the element, not the properties it’s going to set. Also, no deep property access either.